packages/manager/modules/support/src/tickets/new-ticket/creation-form/creation-form.controller.js

```diff
@@ -20,12 +19,7 @@ export default class SupportNewCreationFormController {
   }
 
   get subject() {
-    const issue = findLast(this.issues, 'subject');
-    return issue ? issue.subject : undefined;
-  }
-
-  getLastIssue() {
-    return last(this.issues);
+    return get(this.issue, 'subject');
```

Comment:
Since the binding shows that `issue: '<'` (issue can't be null), you can directly `return this.issue.subject` no ?

Reply:
i don't think '<' garantes that the binding will be defined